src/main/kotlin/io/github/gradlenexus/publishplugin/internal/NexusClient.kt

```diff
@@ -97,6 +97,28 @@ class NexusClient(private val baseUrl: URI, username: String?, password: String?
         }
     }
 
+    fun closeStagingRepository(stagingRepositoryId: String) {
+        try {
+            val response = api.closeStagingRepo(Dto(StagingRepositoryToTransit(listOf(stagingRepositoryId), "Closed by io.github.gradle-nexus.publish-plugin Gradle plugin"))).execute()
```

Comment:
I wish there was a more-or-less meaningful (configurable?) comment.

Reply:
Please raise an issue for that. Could be implemented in the future - not an MVP.